client/components/edit/ObjectButtonBar.module.css

```diff
@@ -0,0 +1,18 @@
+.objectBar {
```

Comment:
Indentation looks weird in this file; I think you have a mix of tabs and spaces. Can you standardize on tabs? (Maybe a bad IDE setting?).

Reply:
Sorry, I phrased this badly -- we should standardize our tabs to use spaces. "Standardize on tabs" is not what I meant, and was the worst possible way I could have said that. ;-)

Reply:
Fixed

Reply:
There were still a lot of tabs in the diffs introduced here. I just did a global search and replace to change \t to four spaces, resulting in commit c503f4238a0f22ca01e0d13cb72f9561955068e1 -- hopefully that has taken care of everything.